src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/ServiceOfNotice.cshtml

```diff
@@ -0,0 +1,105 @@
+﻿@using EA.Prsd.Core.Web.Mvc.RazorHelpers
+@using EA.Weee.Core.DataStandards
+@using EA.Weee.Web.Areas.Producer.Controllers
+@using MvcNavigationHelpers
+@using EA.Weee.Web.RazorHelpers
+@model EA.Weee.Web.Areas.Producer.ViewModels.ServiceOfNoticeViewModel
+
+@{
+    const string title = "Service of notice";
+    ViewBag.Title = title + " - GOV.UK";
+}
+@section full_main_content
+{
+    @(this.WeeeGds().BackLink(@Url.UrlFor<EA.Weee.Web.Areas.Producer.Controllers.ProducerController>(a => a.TaskList())))
+
+    <span class="govuk-caption-xl govuk-!-padding-bottom-3">Step 3 of 5</span>
+
+    <header class="hgroup">
+        <h1 class="govuk-heading-l">@title</h1>
+    </header>
+
+    <p>
+        Provide your address for the service of notices. <br /><br />
+
+        For authorised representatives of non-UK established producers this will be your preferred address.
+    </p>
+
+    @using (Html.BeginForm("ServiceOfNotice", "ProducerSubmission", FormMethod.Get, new { id = "sameAsOrganisationAddressForm" }))
+    {
+        @Html.Hidden("copyAddress", true)
+
+        <div class="govuk-form-group">
+
+            @Html.Gds().CheckBoxFrontEndFor(m => m.SameAsOrganisationAddress, new { @id = "SameAsOrganisationAddress", @class = "govuk-checkboxes__input", onchange = "submitFormOnCheckboxChange()" })
```

Comment:
Updates this to use the GDS control and it seems to work, may just need to stick it through Wave.